File: 250416/배열의 특정 위치 찾기/find-specific-location-fo-array.js
const fs = require('fs');
const numbs = fs.readFileSync(0).toString().trim().split(' ').map(Number);

let sum1 = 0;
for (let i=1; i<10; i+=2) {
    sum1 += numbs[i];
}

let sum2 = 0;
for (let i=2; i<10; i+=3) {
    sum2 += numbs[i];
}

console.log(sum1, (sum2/3).toFixed(1));

File: 250416/배열의 특정 위치 찾기/find-specific-location-fo-array.py
arr = list(map(int, input().split()))

sum1 = 0
for i in range(1, 10, 2):
    sum1 += arr[i]

sum2 = 0
for i in range(2, 10, 3):
    sum2 += arr[i]

print(f'{sum1} {sum2/3:.1f}')